README.md

```diff
@@ -504,6 +504,56 @@ class StaticBitIterator implements BitIterator {
 }
 ```
 
+## SSL/TLS
+
+Make sure the Pilosa server runs on a TLS address. [How To Set Up a Secure Cluster](https://www.pilosa.com/docs/latest/tutorials/#how-to-set-up-a-secure-cluster) tutorial explains how to do that.
+
+In order to enable TLS support on the client side, scheme of the address should be `https` which should be explicitly specified, e.g.: `https://01.pilosa.local:10501`
```

Comment:
"... client side, the scheme of the address should be explicity specified as `https`, e.g.: ..."